pkg/kubelet/status/status_manager.go

```diff
@@ -675,9 +675,6 @@ func (m *manager) updateStatusInternal(pod *v1.Pod, status v1.PodStatus, forceUp
 	// Set PodReadyToStartContainersCondition.LastTransitionTime.
 	updateLastTransitionTime(&status, &oldStatus, v1.PodReadyToStartContainers)
 
-	// Set PodScheduledCondition.LastTransitionTime.
-	updateLastTransitionTime(&status, &oldStatus, v1.PodScheduled)
```

Comment:
do we need to keep it as a safeguard? Or can it cause troubles? Sorry, just glancing over the PR for now.